tests/integration/security/testdata/authz/v1beta1-conditions.yaml.tmpl

```diff
@@ -118,28 +98,22 @@ spec:
   rules:
   - to:
     - operation:
-        paths: ["/source-namespace-a"]
+        paths: ["/source-namespace-a", 'regex:\/source-namespace-a-regexp\/[0-9]?']
     when:
     - key: source.namespace
       values: ["{{ .NamespaceA }}"]
+  - to:
+    - operation:
+        paths: ["/source-namespace-a-regex-value"]
+    when:
+    - key: source.namespace
+      values: ["regex:{{ .NamespaceA }}"]
   - to:
     - operation:
         paths: ["/source-namespace-b"]
     when:
     - key: source.namespace
       values: ["{{ .NamespaceB }}"]
----
-
-apiVersion: security.istio.io/v1beta1
```

Comment:
why are you removing this and merging them to a single policy? you can just add your regex to existing policy or better just add a new policy for regex.

Reply:
It seems simplified the yaml file. Do you prefer I keep original policy or merge them together?

Reply:
Sure I will recover to original policy. Thanks!